src/components/Filter/Filter.tsx

```diff
@@ -0,0 +1,26 @@
+import cn from 'classnames';
+import { useContext } from 'react';
+import { FilterType } from '../../types/FilterType';
+import { FilterContext } from '../../contexts/FilterContext';
+
+export const Filter: React.FC = () => {
+  const { filter: filterType, setFilter } = useContext(FilterContext);
+
+  return (
+    <nav className="filter">
+      {Object.values(FilterType).map(filter => (
+        <a
+          key={filter}
+          href={`#/${filter === FilterType.All
```

Comment:
split this with variable
```suggestion
          href={...}
```